### ISSUE TITLE
[Survey] Pressing the button to create a Survey while the Surveys index is loading throws an error.

### ISSUE THREAD

**Issue by mmuller** [instedd/ask#895]
Found in version: aebfaec

Reproduce steps:
Navigate to the Surveys index.
When the page is still loading press the button to create a Survey.

Obtained:
An unexpected error is thrown.
`Sorry, something went wrong` is displayed.

Additional information:
https://sentry.io/instedd/ask-stg/issues/251694903/
https://sentry.io/instedd/ask-stg/issues/229902951/

**Comment by asterite** [comment]
@mmuller I can't reproduce this. When you enter the survey list, the "+" button only appears once the surveys are shown. Additionally, the error messages in Sentry seem to be a totally different error. Could you try to record it with http://recordit.co and upload it here?

**Comment by mmuller** [comment]
@asterite I could reproduce it with @fgasperij yesterday, but I can't reproduce this issue today.

**Comment by mmuller** [comment]
@asterite finally I could obtain the video: http://recordit.co/nsAeRdIMeJ

**Comment by mmuller** [comment]
From browser's console:

```
error: TypeError: Cannot read property 'tweensContainer' of undefined at e (https://ask-stg.instedd.org/js/app-c6d5c7ab772366c45c67d5c73e430ebd.js?vsn=d:91:14151) at t (https://ask-stg.instedd.org/js/app-c6d5c7ab772366c45c67d5c73e430ebd.js?vsn=d:54:22898)
message
:
"Cannot read property 'tweensContainer' of undefined"
stack
:
"TypeError: Cannot read property 'tweensContainer' of undefined↵    at e (https://ask-stg.instedd.org/js/app-c6d5c7ab772366c45c67d5c73e430ebd.js?vsn=d:91:14151)↵    at t (https://ask-stg.instedd.org/js/app-c6d5c7ab772366c45c67d5c73e430ebd.js?vsn=d:54:22898)"
```

**Comment by waj** [comment]
Seems to be something related with an animation: https://github.com/Dogfalo/materialize/issues/1031

Apparently, this can happen when an element being animated is removed from the DOM.

**Comment by asterite** [comment]
Do we really need to keep this floating around for 0.9? I consider it a **tiny** issue, hardly reproducible.

**Comment by nekron** [comment]
If we can identify what component is failing to be removed (I think this could be an issue with the `ConfirmationModal`), we can hook in the react callback `componentWillUnmount` of the component and do:
```    $('#element').velocity("finish").velocity("stop")
    $('#element').remove()```

**Comment by asterite** [comment]
Well, good luck reproducing the issue, because one has to reproduce it to know if any JS change will fix it :-)

**Comment by asterite** [comment]
I was just able to reproduce this. New steps to reproduce:

1. Create a project
2. Create about 200 surveys in that project
3. Reload the surveys index page.
4. Just when the page finishes reloading, quickly click the "+" button

There's a chance (because it doesn't always happen) that the error message will appear in the console.

**Comment by asterite** [comment]
Another note: this is super hard to reproduce in development mode (in fact I wasn't able to, I only reproduced it in staging) because loading times are slower there, which makes it harder to fix.

The fix for ConfirmationModal makes no sense, there's no `remove()` call there. The only `remove()` calls I found were in the materialize library, it should be fixed in the library. Unfortunately the maintainers closed the issue for no apparent reason.
